src/code/z_demo.c

```diff
@@ -175,85 +197,93 @@ void func_800645A0(PlayState* play, CutsceneContext* csCtx) {
     }
 
     if (gSaveContext.cutsceneIndex >= 0xFFF0) {
-        func_80068ECC(play, csCtx);
-        sCsStateHandlers2[csCtx->state](play, csCtx);
+        Cutscene_SetupScripted(play, csCtx);
+        sScriptedCutsceneHandlers[csCtx->state](play, csCtx);
     }
 }
 
-void func_80064720(PlayState* play, CutsceneContext* csCtx) {
+void CutsceneHandler_DoNothing(PlayState* play, CutsceneContext* csCtx) {
 }
 
-u32 func_8006472C(PlayState* play, CutsceneContext* csCtx, f32 target) {
-    return Math_StepToF(&csCtx->unk_0C, target, 0.1f);
+u32 Cutscene_StepTimer(PlayState* play, CutsceneContext* csCtx, f32 target) {
+    return Math_StepToF(&csCtx->timer, target, 0.1f);
 }
 
-void func_80064760(PlayState* play, CutsceneContext* csCtx) {
+void CutsceneHandler_StartManual(PlayState* play, CutsceneContext* csCtx) {
```

Comment:
this may just be me being illiterate but I was very confused for a few minutes between this and `Cutscene_StartManual` before I realized they were two different functions

I don't have a suggestion though, maybe later with a clearer view of how the functions play together

Reply:
maybe but I'm not a fan:
 swap the words, as if manual/script(ed) was a subsystem, i.e. `Cutscene_StartManual` and `CutsceneHandler_ManualStart` (but again, bleh)

Reply:
this function being `CutsceneHandler_` is supposed to indicate that its the handler function that actually processes things

`Cutscene_StartManual` is what you call in code to indicate that you are about to start a manual cutscene. it sets the state to start (which will then cause `CutsceneHandler_` one to run)

Reply:
I think this issue will be alleviated when proper headers exist and it exposes what is expected to be called externally all in one place

this confusion only exists when reading the file from top to bottom, it wont really be a problem for people using the system. and I think the naming scheme makes sense as-is, as long as you notice the system name difference lol

Reply:
right, `CutsceneHandler_` stuff is private
I guess that address the issue (that with "just don't be blind 4Head")